shared/src/main/java/uk/gov/di/authentication/shared/entity/EmailCheckResultSqsMessage.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.di.authentication.shared.entity;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import uk.gov.di.authentication.shared.validation.Required;
+
+public class EmailCheckResultSqsMessage {
```

Comment:
Just wondering if we'd be able to do this as a record? I think we can't for dynamo db classes at the moment because as far as I can tell the dynamo library doesn't support them, but gson does and I'm assuming that's the only thing that matters in this case.

Reply:
Yeah, good point. Thanks @BeckaL